public/templates/book.php

```diff
@@ -1,31 +1,34 @@
 <?php
 
-use App\CreationLog;
 use App\FontProvider;
 use App\GeneratorSelector;
 use App\Util\Api;
 
 include 'header.php';
 ?>
+<ol class="breadcrumb">
```

Comment:
The 'breadcrumb' is used to show the current position in a hierarchy, not a navigation menu. I think it could just be left out here, because there's already a link to the statisics in the footer.

Reply:
Breakcrumb removed, thank you for the clarification :)